frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -98,13 +107,72 @@ export type InfoControlProps = {
  */
 const InfoControl: React.FC<InfoControlProps> = ({ open, setOpen, onHeaderActionClick }) => {
   const popUpContext = React.useContext(PropertyPopUpContext);
+  const leaflet = useLeaflet();
+  const propertyInfo = popUpContext.propertyInfo;
+  const location = useLocation();
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const { getParcel } = useApi();
+  const jumpToView = () =>
```

Comment:
moved some of this action logic up a level since the associatedBuilding link needs it as well.